2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],3)==[85, 75, 65] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],2)==[85, 75] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],5)==[85, 75, 65, 58, 35] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{ "O": "Orange", "W": "White", "B": "Black" })=={'B': 'Black', 'R': 'Red', 'P': 'Pink', 'G': 'Green', 'W': 'White', 'O': 'Orange'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{"L":"lavender","B":"Blue"})=={'W': 'White', 'P': 'Pink', 'B': 'Black', 'R': 'Red', 'G': 'Green', 'L': 'lavender'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" },{"L":"lavender","B":"Blue"},{ "G": "Green", "W": "White" })=={'B': 'Black', 'P': 'Pink', 'R': 'Red', 'G': 'Green', 'L': 'lavender', 'W': 'White'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([12,3,5,7,19], 2) == 3 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([17,24,8,23], 3) == 8 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([16,21,25,36,4], 4) == 36 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([25, 24, 15, 4, 5, 29, 110],[19, 20, 11, 56, 25, 233, 154],[24, 26, 54, 48])==[4, 5, 11, 15, 19, 20, 24, 24, 25, 25, 26, 29, 48, 54, 56, 110, 154, 233] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([1, 3, 5, 6, 8, 9], [2, 5, 7, 11], [1, 4, 7, 8, 12])==[1, 1, 2, 3, 4, 5, 5, 6, 7, 7, 8, 8, 9, 11, 12] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([18, 14, 10, 9, 8, 7, 9, 3, 2, 4, 1],[25, 35, 22, 85, 14, 65, 75, 25, 58],[12, 74, 9, 50, 61, 41])==[1, 2, 3, 4, 7, 8, 9, 9, 9, 10, 12, 14, 14, 18, 22, 25, 25, 35, 41, 50, 58, 61, 65, 74, 75, 85] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,2,2,3]) == 3 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,3,3,4,4,5,5,7,7,8]) == 8 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,2,2,3,3,4,4]) == 1 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(2, 3, 7) == (2, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(4, 2, 7) == None 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(1, 13, 17) == (4, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([5, 4, 7, 2, 1]) == 4 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([7, 2, 8, 1, 0, 5, 11]) == 9 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([1, 2, 3]) == 1 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(1) == (1.0, 0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(4) == (4.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(5) == (5.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,8)==10.63014581273465 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(3,4)==5 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,15)==16.55294535724685 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,50)==("HELLO", 5, [50], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,100)==(("HELLO", 5, [100],True)) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,500)==("HELLO", 5, [500], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],3)==[60, 54, 50] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],4)==[60, 54, 50, 48] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],5)==[60, 54, 50, 48, 45] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(10) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(12) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(15) == 15 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(25,55,65)==55.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(20,10,30)==20.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(15,45,75)==45.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'r')==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'5')==False 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c","e"),3)==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3,4,5],[6,7,8,9]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3],[4,5,6]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,4,5],[1,4,5]) == True 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(5,3,2)==-198 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(9,8,4)==-2336 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(2,4,6)==-130 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(15,25,35)==20 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(10,20,30)==15 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(6,9,4)==7.5 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 3 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 3), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 2 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 2), (6, 5), (2, 1)] ) == 4 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(5)==150 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(3)==54 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(10)==600 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(1500,1200)==0 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(100,200)==100 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(2000,5000)==3000 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,20)==200 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,5)==50 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(4,2)==8 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(3, 4), (5, 6), (9, 10), (4, 5)] , [(5, 4), (3, 4), (6, 5), (9, 11)]) == {(4, 5), (3, 4), (5, 6)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(4, 1), (7, 4), (11, 13), (17, 14)] , [(1, 4), (7, 4), (16, 12), (10, 13)]) == {(4, 7), (1, 4)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(2, 1), (3, 2), (1, 3), (1, 4)] , [(11, 2), (2, 3), (6, 2), (1, 3)]) == {(1, 3), (2, 3)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([5,4,7,2,1],5) == 6 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([7,2,8,1,0,5,11],7) == 12 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([1,2,3],3) == 2 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-2, -3, 4, -1, -2, 1, 5, -3], 8) == 7 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-3, -4, 5, -2, -3, 2, 6, -4], 8) == 8 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-4, -5, 6, -3, -4, 3, 7, -5], 8) == 10 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,1],3) == 2 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,1,1,1],4) == 0 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,3,4,5],5) == 10 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

302 assert izip(("A", 1, "B", 2, "C", 3) ) == sys.getsizeof(("A", 1, "B", 2, "C", 3)) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

302 assert izip((1, "Raju", 2, "Nikhil", 3, "Deepanshu") ) == sys.getsizeof((1, "Raju", 2, "Nikhil", 3, "Deepanshu")) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([4,5,6],[1, 2, 3])==[4.0,2.5,2.0] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([3,2],[1,4])==[3.0, 0.5] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([90,120],[50,70])==[1.8, 1.7142857142857142] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([1, 12, 15, 26, 38], [2, 13, 17, 30, 45], 5) == 16.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([2, 4, 8, 9], [7, 13, 19, 28], 4) == 8.5 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([3, 6, 14, 23, 36, 42], [2, 18, 27, 39, 49, 55], 6) == 25.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(-1) == None 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(0) == 0 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(2) == 4 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([5,9,7,6],4) == 47 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3,5],3) == 12 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3],2) == 4 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},6.0,70)=={'Cierra Vega': (6.2, 70)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.9,67)=={'Cierra Vega': (6.2, 70),'Kierra Gentry': (6.0, 68)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.7,64)=={'Cierra Vega': (6.2, 70),'Alden Cantrell': (5.9, 65),'Kierra Gentry': (6.0, 68),'Pierre Cox': (5.8, 66)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((1, 5, 7, 8, 10)) == (5, 35, 56, 80) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((2, 4, 5, 6, 7)) == (8, 20, 30, 42) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12, 13, 14, 9, 15)) == (156, 182, 126, 135) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12,)) == () 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([10,20,30],[15,25,35])==[25,45,65] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([1,2,3],[5,6,7])==[6,8,10] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([15,20,30],[15,45,75])==[30,65,105] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(3)==15.588457268119894 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(20)==692.8203230275509 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(10)==173.20508075688772 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('Manjeet', 10), ('Akshat', 4), ('Akash', 2), ('Nikhil', 8)], 2) == [('Akash', 2), ('Akshat', 4)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('Sanjeev', 11), ('Angat', 5), ('Akash', 3), ('Nepin', 9)], 3) == [('Akash', 3), ('Angat', 5), ('Nepin', 9)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('tanmay', 14), ('Amer', 11), ('Ayesha', 9), ('SKD', 16)], 1) == [('Ayesha', 9)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((1,5,3,19,18,25),6) == 1 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((4,3,2,6),4) == 1 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((30,5,20,9),4) == 4 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([3,2,1]) == 1 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([4,5,1,2,3]) == 2 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([7,8,9,1,2,3]) == 3 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([1,2,3]) == 0 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([1,3,2]) == 2 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],3)==[85, 75, 65] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],2)==[85, 75] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],5)==[85, 75, 65, 58, 35] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{ "O": "Orange", "W": "White", "B": "Black" })=={'B': 'Black', 'R': 'Red', 'P': 'Pink', 'G': 'Green', 'W': 'White', 'O': 'Orange'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{"L":"lavender","B":"Blue"})=={'W': 'White', 'P': 'Pink', 'B': 'Black', 'R': 'Red', 'G': 'Green', 'L': 'lavender'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" },{"L":"lavender","B":"Blue"},{ "G": "Green", "W": "White" })=={'B': 'Black', 'P': 'Pink', 'R': 'Red', 'G': 'Green', 'L': 'lavender', 'W': 'White'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([12,3,5,7,19], 2) == 3 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([17,24,8,23], 3) == 8 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([16,21,25,36,4], 4) == 36 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([25, 24, 15, 4, 5, 29, 110],[19, 20, 11, 56, 25, 233, 154],[24, 26, 54, 48])==[4, 5, 11, 15, 19, 20, 24, 24, 25, 25, 26, 29, 48, 54, 56, 110, 154, 233] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([1, 3, 5, 6, 8, 9], [2, 5, 7, 11], [1, 4, 7, 8, 12])==[1, 1, 2, 3, 4, 5, 5, 6, 7, 7, 8, 8, 9, 11, 12] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([18, 14, 10, 9, 8, 7, 9, 3, 2, 4, 1],[25, 35, 22, 85, 14, 65, 75, 25, 58],[12, 74, 9, 50, 61, 41])==[1, 2, 3, 4, 7, 8, 9, 9, 9, 10, 12, 14, 14, 18, 22, 25, 25, 35, 41, 50, 58, 61, 65, 74, 75, 85] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,2,2,3]) == 3 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,3,3,4,4,5,5,7,7,8]) == 8 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,2,2,3,3,4,4]) == 1 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(2, 3, 7) == (2, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(4, 2, 7) == None 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(1, 13, 17) == (4, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([5, 4, 7, 2, 1]) == 4 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([7, 2, 8, 1, 0, 5, 11]) == 9 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([1, 2, 3]) == 1 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(1) == (1.0, 0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(4) == (4.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(5) == (5.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,8)==10.63014581273465 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(3,4)==5 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,15)==16.55294535724685 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,50)==("HELLO", 5, [50], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,100)==(("HELLO", 5, [100],True)) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,500)==("HELLO", 5, [500], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],3)==[60, 54, 50] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],4)==[60, 54, 50, 48] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],5)==[60, 54, 50, 48, 45] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(10) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(12) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(15) == 15 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(25,55,65)==55.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(20,10,30)==20.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(15,45,75)==45.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'r')==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'5')==False 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c","e"),3)==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3,4,5],[6,7,8,9]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3],[4,5,6]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,4,5],[1,4,5]) == True 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(5,3,2)==-198 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(9,8,4)==-2336 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(2,4,6)==-130 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(15,25,35)==20 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(10,20,30)==15 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(6,9,4)==7.5 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 3 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 3), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 2 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 2), (6, 5), (2, 1)] ) == 4 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(5)==150 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(3)==54 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(10)==600 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(1500,1200)==0 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(100,200)==100 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(2000,5000)==3000 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,20)==200 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,5)==50 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(4,2)==8 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(3, 4), (5, 6), (9, 10), (4, 5)] , [(5, 4), (3, 4), (6, 5), (9, 11)]) == {(4, 5), (3, 4), (5, 6)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(4, 1), (7, 4), (11, 13), (17, 14)] , [(1, 4), (7, 4), (16, 12), (10, 13)]) == {(4, 7), (1, 4)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(2, 1), (3, 2), (1, 3), (1, 4)] , [(11, 2), (2, 3), (6, 2), (1, 3)]) == {(1, 3), (2, 3)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([5,4,7,2,1],5) == 6 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([7,2,8,1,0,5,11],7) == 12 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([1,2,3],3) == 2 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-2, -3, 4, -1, -2, 1, 5, -3], 8) == 7 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-3, -4, 5, -2, -3, 2, 6, -4], 8) == 8 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-4, -5, 6, -3, -4, 3, 7, -5], 8) == 10 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,1],3) == 2 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,1,1,1],4) == 0 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,3,4,5],5) == 10 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

302 assert izip(("A", 1, "B", 2, "C", 3) ) == sys.getsizeof(("A", 1, "B", 2, "C", 3)) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

302 assert izip((1, "Raju", 2, "Nikhil", 3, "Deepanshu") ) == sys.getsizeof((1, "Raju", 2, "Nikhil", 3, "Deepanshu")) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([4,5,6],[1, 2, 3])==[4.0,2.5,2.0] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([3,2],[1,4])==[3.0, 0.5] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([90,120],[50,70])==[1.8, 1.7142857142857142] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([1, 12, 15, 26, 38], [2, 13, 17, 30, 45], 5) == 16.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([2, 4, 8, 9], [7, 13, 19, 28], 4) == 8.5 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([3, 6, 14, 23, 36, 42], [2, 18, 27, 39, 49, 55], 6) == 25.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(-1) == None 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(0) == 0 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(2) == 4 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([5,9,7,6],4) == 47 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3,5],3) == 12 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3],2) == 4 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},6.0,70)=={'Cierra Vega': (6.2, 70)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.9,67)=={'Cierra Vega': (6.2, 70),'Kierra Gentry': (6.0, 68)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.7,64)=={'Cierra Vega': (6.2, 70),'Alden Cantrell': (5.9, 65),'Kierra Gentry': (6.0, 68),'Pierre Cox': (5.8, 66)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((1, 5, 7, 8, 10)) == (5, 35, 56, 80) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((2, 4, 5, 6, 7)) == (8, 20, 30, 42) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12, 13, 14, 9, 15)) == (156, 182, 126, 135) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12,)) == () 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([10,20,30],[15,25,35])==[25,45,65] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([1,2,3],[5,6,7])==[6,8,10] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([15,20,30],[15,45,75])==[30,65,105] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(3)==15.588457268119894 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(20)==692.8203230275509 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(10)==173.20508075688772 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('Manjeet', 10), ('Akshat', 4), ('Akash', 2), ('Nikhil', 8)], 2) == [('Akash', 2), ('Akshat', 4)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('Sanjeev', 11), ('Angat', 5), ('Akash', 3), ('Nepin', 9)], 3) == [('Akash', 3), ('Angat', 5), ('Nepin', 9)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

375 assert ik([('tanmay', 14), ('Amer', 11), ('Ayesha', 9), ('SKD', 16)], 1) == [('Ayesha', 9)] 
typetest.py:16: error: Too many arguments for "ik"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((1,5,3,19,18,25),6) == 1 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((4,3,2,6),4) == 1 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

384 assert ia_min_diff((30,5,20,9),4) == 4 
typetest.py:15: error: Too many arguments for "ia_min_diff"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([3,2,1]) == 1 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([4,5,1,2,3]) == 2 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([7,8,9,1,2,3]) == 3 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([1,2,3]) == 0 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

419 assert ____rotations([1,3,2]) == 2 
typetest.py:11: error: Missing positional argument "n" in call to "____rotations"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],3)==[85, 75, 65] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],2)==[85, 75] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

2 assert ileven( [25, 35, 22, 85, 14, 65, 75, 22, 58],5)==[85, 75, 65, 58, 35] 
typetest.py:10: error: Too many arguments for "ileven"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{ "O": "Orange", "W": "White", "B": "Black" })=={'B': 'Black', 'R': 'Red', 'P': 'Pink', 'G': 'Green', 'W': 'White', 'O': 'Orange'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" }, { "G": "Green", "W": "White" },{"L":"lavender","B":"Blue"})=={'W': 'White', 'P': 'Pink', 'B': 'Black', 'R': 'Red', 'G': 'Green', 'L': 'lavender'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

41 assert izmerge({ "R": "Red", "B": "Black", "P": "Pink" },{"L":"lavender","B":"Blue"},{ "G": "Green", "W": "White" })=={'B': 'Black', 'P': 'Pink', 'R': 'Red', 'G': 'Green', 'L': 'lavender', 'W': 'White'} 
typetest.py:15: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([12,3,5,7,19], 2) == 3 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([17,24,8,23], 3) == 8 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

55 assert ikth([16,21,25,36,4], 4) == 36 
typetest.py:13: error: Missing positional argument "k" in call to "ikth"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([25, 24, 15, 4, 5, 29, 110],[19, 20, 11, 56, 25, 233, 154],[24, 26, 54, 48])==[4, 5, 11, 15, 19, 20, 24, 24, 25, 25, 26, 29, 48, 54, 56, 110, 154, 233] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([1, 3, 5, 6, 8, 9], [2, 5, 7, 11], [1, 4, 7, 8, 12])==[1, 1, 2, 3, 4, 5, 5, 6, 7, 7, 8, 8, 9, 11, 12] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

61 assert izmerge([18, 14, 10, 9, 8, 7, 9, 3, 2, 4, 1],[25, 35, 22, 85, 14, 65, 75, 25, 58],[12, 74, 9, 50, 61, 41])==[1, 2, 3, 4, 7, 8, 9, 9, 9, 10, 12, 14, 14, 18, 22, 25, 25, 35, 41, 50, 58, 61, 65, 74, 75, 85] 
typetest.py:14: error: Too many arguments for "izmerge"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,2,2,3]) == 3 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,1,3,3,4,4,5,5,7,7,8]) == 8 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

69 assert ix([1,2,2,3,3,4,4]) == 1 
typetest.py:26: error: Missing positional argument "x" in call to "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(2, 3, 7) == (2, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(4, 2, 7) == None 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

91 assert xtest(1, 13, 17) == (4, 1) 
typetest.py:14: error: Too many arguments for "xtest"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([5, 4, 7, 2, 1]) == 4 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([7, 2, 8, 1, 0, 5, 11]) == 9 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

97 assert ixor([1, 2, 3]) == 1 
typetest.py:9: error: Missing positional argument "y" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(1) == (1.0, 0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(4) == (4.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

128 assert xto_polar(5) == (5.0,0.0) 
typetest.py:13: error: Missing positional argument "y" in call to "xto_polar"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,8)==10.63014581273465 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(3,4)==5 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

163 assert xtri(7,15)==16.55294535724685 
typetest.py:12: error: Missing positional argument "c" in call to "xtri"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,50)==("HELLO", 5, [50], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,100)==(("HELLO", 5, [100],True)) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

174 assert ix(("HELLO", 5, [], True) ,2,500)==("HELLO", 5, [500], True) 
typetest.py:8: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],3)==[60, 54, 50] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],4)==[60, 54, 50, 48] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

175 assert ile_max([1, 2, 3, 4, 5, 6],[3, 6, 8, 9, 10, 6],5)==[60, 54, 50, 48, 45] 
typetest.py:15: error: Too many arguments for "ile_max"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(10) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(12) == 14 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

178 assert ixor(15) == 15 
typetest.py:9: error: Missing positional argument "b" in call to "ixor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(25,55,65)==55.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(20,10,30)==20.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

189 assert ____median_three_numbers(15,45,75)==45.0 
typetest.py:16: error: Too many arguments for "____median_three_numbers"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'r')==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c", "e"),'5')==False 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

195 assert ix(("w", 3, "r", "e", "s", "o", "u", "r", "c","e"),3)==True 
typetest.py:13: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3,4,5],[6,7,8,9]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,2,3],[4,5,6]) == False 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

204 assert ivalue([1,4,5],[1,4,5]) == True 
typetest.py:13: error: Too many arguments for "ivalue"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(5,3,2)==-198 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(9,8,4)==-2336 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

218 assert ia_parabola(2,4,6)==-130 
typetest.py:12: error: Too many arguments for "ia_parabola"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(15,25,35)==20 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(10,20,30)==15 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

220 assert ia_trapezium(6,9,4)==7.5 
typetest.py:18: error: Too many arguments for "ia_trapezium"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 3 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 3), (6, 5), (9, 1), (6, 5), (2, 1)] ) == 2 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

226 assert ids([(5, 6), (1, 2), (6, 5), (9, 2), (6, 5), (2, 1)] ) == 4 
typetest.py:9: error: Missing positional argument "b" in call to "ids"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(5)==150 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(3)==54 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

229 assert ia(10)==600 
typetest.py:9: error: Missing positional arguments "y", "z" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(1500,1200)==0 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(100,200)==100 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

239 assert iva(2000,5000)==3000 
typetest.py:12: error: Too many arguments for "iva"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,20)==200 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(10,5)==50 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

245 assert ia(4,2)==8 
typetest.py:9: error: Missing positional argument "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(3, 4), (5, 6), (9, 10), (4, 5)] , [(5, 4), (3, 4), (6, 5), (9, 11)]) == {(4, 5), (3, 4), (5, 6)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(4, 1), (7, 4), (11, 13), (17, 14)] , [(1, 4), (7, 4), (16, 12), (10, 13)]) == {(4, 7), (1, 4)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

257 assert ix([(2, 1), (3, 2), (1, 3), (1, 4)] , [(11, 2), (2, 3), (6, 2), (1, 3)]) == {(1, 3), (2, 3)} 
typetest.py:9: error: Too many arguments for "ix"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([5,4,7,2,1],5) == 6 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([7,2,8,1,0,5,11],7) == 12 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

266 assert xto([1,2,3],3) == 2 
typetest.py:14: error: Too many arguments for "xto"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-2, -3, 4, -1, -2, 1, 5, -3], 8) == 7 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-3, -4, 5, -2, -3, 2, 6, -4], 8) == 8 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

269 assert ile_sublist([-4, -5, 6, -3, -4, 3, 7, -5], 8) == 10 
typetest.py:14: error: Too many arguments for "ile_sublist"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,1],3) == 2 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,1,1,1],4) == 0 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

273 assert ivalid_pairs([1,2,3,4,5],5) == 10 
typetest.py:18: error: Too many arguments for "ivalid_pairs"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

302 assert izip(("A", 1, "B", 2, "C", 3) ) == sys.getsizeof(("A", 1, "B", 2, "C", 3)) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

302 assert izip((1, "Raju", 2, "Nikhil", 3, "Deepanshu") ) == sys.getsizeof((1, "Raju", 2, "Nikhil", 3, "Deepanshu")) 
typetest.py:8: error: Name "sys" is not defined  [name-defined]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([4,5,6],[1, 2, 3])==[4.0,2.5,2.0] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([3,2],[1,4])==[3.0, 0.5] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

321 assert izquierda([90,120],[50,70])==[1.8, 1.7142857142857142] 
typetest.py:12: error: Too many arguments for "izquierda"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([1, 12, 15, 26, 38], [2, 13, 17, 30, 45], 5) == 16.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([2, 4, 8, 9], [7, 13, 19, 28], 4) == 8.5 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

324 assert izmedian([3, 6, 14, 23, 36, 42], [2, 18, 27, 39, 49, 55], 6) == 25.0 
typetest.py:18: error: Too many arguments for "izmedian"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(-1) == None 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(0) == 0 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

328 assert ia(2) == 4 
typetest.py:11: error: Missing positional arguments "b", "c" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([5,9,7,6],4) == 47 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3,5],3) == 12 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

335 assert xtor([7,3],2) == 4 
typetest.py:14: error: Too many arguments for "xtor"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},6.0,70)=={'Cierra Vega': (6.2, 70)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.9,67)=={'Cierra Vega': (6.2, 70),'Kierra Gentry': (6.0, 68)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

346 assert ile_minimal({'Cierra Vega': (6.2, 70), 'Alden Cantrell': (5.9, 65), 'Kierra Gentry': (6.0, 68), 'Pierre Cox': (5.8, 66)},5.7,64)=={'Cierra Vega': (6.2, 70),'Alden Cantrell': (5.9, 65),'Kierra Gentry': (6.0, 68),'Pierre Cox': (5.8, 66)} 
typetest.py:13: error: Too many arguments for "ile_minimal"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((1, 5, 7, 8, 10)) == (5, 35, 56, 80) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((2, 4, 5, 6, 7)) == (8, 20, 30, 42) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12, 13, 14, 9, 15)) == (156, 182, 126, 135) 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

350 assert xtuple((12,)) == () 
typetest.py:12: error: Missing positional argument "t_2" in call to "xtuple"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([10,20,30],[15,25,35])==[25,45,65] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([1,2,3],[5,6,7])==[6,8,10] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

351 assert ia_plus_ib([15,20,30],[15,45,75])==[30,65,105] 
typetest.py:9: error: Missing positional arguments "a_n", "b_n" in call to "ia_plus_ib"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(3)==15.588457268119894 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

364 assert ia(20)==692.8203230275509 
typetest.py:8: error: Missing positional arguments "b", "c", "d" in call to "ia"  [call-arg]
Found 1 error in 1 file (checked 1 source file)

